### DISCUSSION TITLE
MYKI - Removed?

### DISCUSSION THREAD

**Discussion by kaganergovan** [elrumo/macOS_Big_Sur_icons_replacements#1204] [Icon request] [answered]
Hey Dear Community,

There was MYKI icon on site but I can't find it right now. Is it deleted? If deleted, could anyone upload this?

Regards,

**Comment by OMGzilla-git** [comment, marked as answer]
I have no idea what the one you had previously looked like, but I just made a new one and submitted it. It should get approved sometime tomorrow.

**Reply by kaganergovan** [reply]
Looking forward to download it! Thanks so much.

**Reply by kaganergovan** [reply]
@OMGzilla-git just downloaded and looks perfect. Thanks!
